### PR TITLE
docs: how to provision datasource

### PR DESCRIPTION
**Pull request by dmitryk-dk**
I added documentation on how to provision data source if needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,67 @@ See more about chart settings [here](https://github.com/grafana/helm-charts/blob
 
 Another option would be to build custom Grafana image with plugin based on same installation instructions.
 
+## Plugin provision
```
Comment (hagen1778) [inline README.md:82]:
```suggestion
## Provisioning in Grafana
```

File: README.md [hunk 2]
```diff
@@ -79,6 +79,67 @@ See more about chart settings [here](https://github.com/grafana/helm-charts/blob
...
+
+Provision of grafana plugin requires to:
```
Comment (hagen1778) [inline README.md:84]: I'd expect to have something like this https://github.com/Altinity/clickhouse-grafana#configure-the-datasource-with-provisioning

1. Reference to grafana docs
2. Fields which are important for our plugin, like type

All the extra steps with docker-compose are nice, but not always a case for users.

Comment (dmitryk-dk) [reply README.md]: I checked different data source plugins; many still need examples of creating or checking their provisioning config. So I have updated the doc. Added more clear example of the provision config file and described the steps on how to check the configuration

File: README.md [hunk 3]
```diff
@@ -79,6 +79,96 @@ See more about chart settings [here](https://github.com/grafana/helm-charts/blob
...
+     # <string> Sets the pre-selected datasource for new panels. 
+     # You can set only one default data source per organization.
+     isDefault: false
+```
+
+You can check your configuration by doing the following steps:
+
+1. Create folder `./provisioning/datasource` with datasource example file:
+
+2. Build frontend and backend part of the plugin:
```
Comment (hagen1778) [inline README.md:134]: I'd recommed downloading the specific release instead of building from sources.

Comment (dmitryk-dk) [reply README.md]: Yes, you are right! The release is better to use than building the project. Updated that part of the documentation

File: README.md [hunk 4]
```diff
@@ -79,6 +79,99 @@ See more about chart settings [here](https://github.com/grafana/helm-charts/blob
...
+           - 3000:3000/tcp
+        volumes:
+           - ./victoriametrics-datasource:/var/lib/grafana/plugins/grafana-datasource
+           - ./provisioning:/etc/grafana/provisioning
+```
+
+4. Run docker-compose file: 
+
+```
+docker-compose -f docker-compose.yaml up
```
Comment (zekker6) [inline README.md:166]: `docker-compose.yaml` is a default name for this file so it is not required to specify it as `-f`

Comment (dmitryk-dk) [reply README.md:166]: Yes, we can use just docker-compose up, but I want to show the user what he is running

**Review by zekker6 — APPROVED** [review]
LGTM

**Review by hagen1778 — APPROVED** [review]

**Comment by hagen1778** [comment]
thank you!

**Pull request merged**